app/services/ccms/requestors/case_add_requestor.rb

```diff
@@ -198,53 +198,48 @@ def generate_provider_details(xml)
       end
 
-      def generate_proceeding(xml, application_proceeding_type)
+      def generate_proceeding(xml, proceeding)
         xml.__send__('casebio:Proceeding') do
-          xml.__send__('casebio:ProceedingCaseID', application_proceeding_type.proceeding_case_p_num)
+          xml.__send__('casebio:ProceedingCaseID', proceeding.case_p_num)
           xml.__send__('casebio:Status', 'Draft')
-          xml.__send__('casebio:LeadProceedingIndicator', application_proceeding_type.lead_proceeding)
-          xml.__send__('casebio:ProceedingDetails') { generate_proceeding_type(xml, application_proceeding_type) }
+          xml.__send__('casebio:LeadProceedingIndicator', proceeding.lead_proceeding)
+          xml.__send__('casebio:ProceedingDetails') { generate_proceeding_type(xml, proceeding) }
         end
       end
 
-      def generate_proceeding_type(xml, application_proceeding_type)
-        proceeding_type = application_proceeding_type.proceeding_type
-        xml.__send__('casebio:ProceedingType', proceeding_type.ccms_code)
-        xml.__send__('casebio:ProceedingDescription', proceeding_type.description)
-        xml.__send__('casebio:MatterType', proceeding_type.ccms_matter_code)
-        xml.__send__('casebio:LevelOfService', proceeding_type.default_level_of_service.service_level_number)
+      def generate_proceeding_type(xml, proceeding)
+        xml.__send__('casebio:ProceedingType', proceeding.ccms_code)
+        xml.__send__('casebio:ProceedingDescription', proceeding.description)
+        xml.__send__('casebio:MatterType', proceeding.ccms_matter_code)
+        xml.__send__('casebio:LevelOfService', proceeding.default_level_of_service_level)
         xml.__send__('casebio:Stage', 8) # TODO: CCMS placeholder - This will need reviewing when we add more proceedings and/or matter types
         xml.__send__('casebio:ClientInvolvementType', 'A')
-        xml.__send__('casebio:ScopeLimitations') { generate_scope_limitations(xml, application_proceeding_type) }
+        xml.__send__('casebio:ScopeLimitations') { generate_scope_limitations(xml, proceeding) }
       end
 
-      def generate_scope_limitations(xml, application_proceeding_type)
-        application_proceeding_type.assigned_scope_limitations.each do |limitation|
-          next if application_proceeding_type.substantive_only? && default_df_limitation?(application_proceeding_type.proceeding_type, limitation)
-
-          generate_scope_limitation(xml, limitation)
+      def generate_scope_limitations(xml, proceeding)
+        xml.__send__('casebio:ScopeLimitation') do
+          xml.__send__('casebio:ScopeLimitation', proceeding.substantive_scope_limitation_code)
+          xml.__send__('casebio:ScopeLimitationWording', proceeding.substantive_scope_limitation_description)
+          xml.__send__('casebio:DelegatedFunctionsApply', proceeding.used_delegated_functions_on.present?)
```

Comment:
As above - can you use the `used_delegated_functions?` method on the proceeding here instead of calling `present?`?